src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyleBindableProperty.cs

```diff
@@ -385,6 +385,27 @@ public partial class ViewStyle
             defaultValueCreator: (bindable) => ((ViewStyle)bindable).cornerRadiusPolicy
         );
 
+        /// <summary> Bindable property of BorderlineWidth. Please do not open it. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty BorderlineWidthProperty = BindableProperty.Create(nameof(BorderlineWidth), typeof(float?), typeof(ViewStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            ((ViewStyle)bindable).borderlineWidth = (float?)newValue;
+        }, defaultValueCreator: (bindable) => ((ViewStyle)bindable).borderlineWidth);
```

Comment:
```suggestion
        public static readonly BindableProperty BorderlineWidthProperty = BindableProperty.Create(nameof(BorderlineWidth), typeof(float?), typeof(ViewStyle), null);
```
It is normal code to use bindinable property, I wonder why do you use private field such as `borderlineWith`?

Reply:
If I change this code without private field, that mean "EVERY" codes in `ViewStyleBindableProperyes.cs` need to fix in this patch. I think It will not works in this PR.

Reply:
Ok, I understand, you do like to keep a consistent with others code